schemas/miscellaneous/circle.schema.tpl.json

```diff
@@ -0,0 +1,17 @@
+{
+    "_type": "https://openminds.ebrains.eu/sands/Circle",
+    "_categories": [
+        "mathematicalShapes"
+    ],
+    "required": [
+        "diameter"
+    ],
+    "properties": {
+        "diameter": {
```

Comment:
to follow the same approach across all shapes we should use the parameters measured from the center of the shape. Therefore we need to ask for the radius instead of the diameter

Reply:
Done